lld/test/ELF/arm-rwpi-debug-relocs.s

```diff
@@ -0,0 +1,57 @@
+// REQUIRES: arm
+// RUN: rm -rf %t && split-file %s %t && cd %t
+
+// RUN: llvm-mc -filetype=obj -triple=armv7a asm.s -o obj.o
+// RUN: ld.lld -T lds.ld obj.o -o exe.elf -e main 2>&1 | FileCheck %s --implicit-check-not=warning: --allow-empty
+// RUN: llvm-objdump -D exe.elf | FileCheck --check-prefix=DISASM %s
+
+// DISASM:      Disassembly of section data1:
+// DISASM:      00001000 <rw>:
```

Comment:
LLD tests tend to use 
```
DISASM-LABEL: <rw>:
DISASM-NEXT: 2000: 000004d2
```
We usually omit the address from all but the first line if the rest are sequential, but in this case there is a ... so I don't think this would apply.